### PR TITLE
feat: Fixed Payment Amount

### LINKED ISSUE #809: allow setting fixed payment amount in member settings
- field is disabled if fixed payments are off in circle settings, but any existing value is still shown
  - the message about needing to enable fixed payments is shown outside the field
- if fixed payments are on, the token type is shown next to the field, i.e. [1000 | USDC]
- show the fixed payment amounts in a column in the member table

Depends on #888

### PR DESCRIPTION
**Pull request by zashton**
fixes #809 

### Test Plan

1. Enable Fixed Payments in Circle Settings by selecting a token
2. Go to any contributor and set a value for their fixed payment amount
3. Save and make sure the amount is reflected in the main contributors table.

Alternatively can just run cypress test for createUsers/updateUsers

### CODE REVIEW

File: src/lib/gql/queries.ts [hunk 1]
```diff
@@ -320,6 +320,10 @@ export const getFullCircle = async (
                 address: true,
                 skills: true,
               },
+              user_private: {
+                fixed_payment_token_type: true,
+                fixed_payment_amount: true,
+              },
```
Comment (topocount) [inline src/lib/gql/queries.ts:326]: ah i see that you've solved this schema visibility issue already.

File: src/pages/AdminPage/components.tsx [hunk 2]
```diff
@@ -769,6 +769,18 @@ export const ContributorsTable = ({
...
+                onClick={() => updateOrder('fixed_payment_amount')}
+              >
+                {renderLabel(
+                  `Fixed Payment Amount ${
+                    circle.fixed_payment_token_type ?? '(Disabled)'
+                  }`,
+                  order,
+                  'fixed_payment_amount'
+                )}
+              </Table.HeaderCell>
```
Comment (topocount) [inline src/pages/AdminPage/components.tsx:783]: hide this behind the feature flag for now.

File: scripts/db_seed_test.ts [hunk 3]
```diff
@@ -181,4 +181,24 @@ async function createCircleInOrgButNoDevMember(protocolId: number) {
...
     ],
   });
-}
\ No newline at end of file
+}
+
+async function createFreshOpenEpochDevAdminWithFixedPayment() {
+  const result = await insertMemberships(
+      getMembershipInput(
+          { protocolInput: { name: 'Fresh Open Epoch Admin' },
```
Comment (topocount) [inline scripts/db_seed_test.ts:189]: might want to change this name

File: scripts/db_seed_test.ts [hunk 4]
```diff
@@ -181,4 +181,24 @@ async function createCircleInOrgButNoDevMember(protocolId: number) {
       },
     ],
   });
-}
\ No newline at end of file
+}
+
+async function createFreshOpenEpochDevAdminWithFixedPayment() {
```
Comment (topocount) [inline scripts/db_seed_test.ts:186]:
```suggestion
async function createFreshOpenEpochDevAdminWithFixedPaymentToken() {
```

**Review by topocount — DISMISSED** [review]
aside from the the feature flag masking, this looks good to me!

**Comment by zashton** [comment]
@topocount the vault cypress test seems to be failing, any clue how to resolve it ?

**Review by topocount — APPROVED** [review]

**Pull request merged**